v4l2py/device.py

```diff
@@ -754,13 +768,29 @@ def write(self, data: bytes) -> None:
         self._fobj.write(data)
 
 
+class MenuItem:
+    def __init__(self, item):
+        self.item = item
+        self.index = item.index
+        self.name = item.name.decode()
+
+    def __repr__(self):
+        return f"<{type(self).__name__} index={self.index} name={self.name}>"
+
+
 class Control:
     def __init__(self, device, info):
         self.device = device
         self.info = info
         self.id = self.info.id
         self.name = info.name.decode()
         self.type = ControlType(self.info.type)
+        if self.type == raw.V4L2_CTRL_TYPE_MENU:
```

Comment:
Maybe you could use `ControlType`?
Something like:
```if self.type == ControlType.MENU:```

Reply:
Good point, makes sense. I've changed it as suggested.